### ISSUE TITLE
Odd icons in Properties window in Dark theme

### ISSUE THREAD

**Issue by IDEExperienceTestSolution** [dotnet/winforms#6012]
* .NET Core Version: 
6.0 rtm 21518.52

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**
In dark them, odd icons in Properties window with extra pixilation issues around them

![CustomerOddIcons](https://user-images.githubusercontent.com/90816462/137835366-22beea27-2a85-4b39-b134-b7aab580c0f1.png)
![OddIcons](https://user-images.githubusercontent.com/90816462/137835403-706a1526-ee8c-4c14-b417-669b2e97c221.png)
![PropertyGrid](https://user-images.githubusercontent.com/90816462/137835406-49249b51-ce49-48de-ad4e-2bdf7fb23404.png)

**Expected behavior:**
There should no extra pixilation issues.

**More info:**
Here is the original customer ticket: https://developercommunity.visualstudio.com/t/Odd-icons-in-Properties-window-after-rep/1551634

**Minimal repro:**
Steps to reproduce:
1.	Create a .NET Core WinForms application.
2.	Make sure the VS color theme is Dark.
3.	Go to Properties window to see the icons.

**Comment by lonitra** [comment]
Taking a look at issues revolving around dark mode. I don't see any obvious resolutions/action items for this. @merriemcgaw was this issue discussed? This also does not look like it is ours so maybe it can be closed or transfered, but I could be wrong.

**Comment by merriemcgaw** [comment]
@lonitra this I believe some of the icons are only in VS, in which case we will want to check them and update them there. The mainline icons (grouping/alphabetizing) will need to be updated in this repo. Keep an eye on this one as you start doing your research. I think there will be action in both repos for it.

**Comment by elachlan** [comment]
@Olina-Zhang can your team retest this please. 

I think @dreddy-work and I might have resolved this when we updated some icons when dealing with globing in #8371 and #8358. But I am not too sure.

**Comment by Olina-Zhang** [comment]
@elachlan it still reproduces in the latest VS and .Net build.
<img width="236" alt="image" src="https://github.com/dotnet/winforms/assets/26474449/e1cee135-8a56-4959-be3b-9961a0b84521">

**Comment by elachlan** [comment]
@merriemcgaw The fix for this will need to be team driven, since we are pulling the icons from VS.

**Comment by merriemcgaw** [comment]
@JeremyKuhne do you know if we can use transparent icons in the PropertyGrid allows for transparent colors in the icon files? If so I can find drive getting updated icons on the VS side, even if it requires a .NET Framework fix.

**Comment by RussKie** [comment]
You'll need to engage the VS UI team via [aka.ms/d3triage](https://aka.ms/d3triage).

**Comment by JeremyKuhne** [comment]
@merriemcgaw yes, transparent icons should work. Sort icons are in protected virtual properties so it should be easy for VS to [update them](https://devdiv.visualstudio.com/DevDiv/_git/VS?path=/src/vsip/Packages/Core/PropertyBrowser/PropertyBrowser.cs&version=GBmain&line=2314&lineEnd=2315&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents). The Events tab is also overridden there, so it can also be tweaked.

**Comment by merriemcgaw** [comment]
@Olina-Zhang can you get an issue filed in the internal repo and close this one? I do want this for 17.9 for sure!

**Comment by elachlan** [comment]
@merriemcgaw this issue is for properygridcontrol. We just need the icons from visual studio to fix it, if it's transparent in visual studio.

**Comment by Olina-Zhang** [comment]
Filed a AzDo bug: https://devdiv.visualstudio.com/DevDiv/_queries/edit/1914940 and close this one.
